.github/workflows/ci.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           cache: 'yarn'
       - name: Install Dependencies
         run: |
-          npm install --global --force yarn@1.22.19
```

Comment:
The version value is used in multipled places, so why not put it in env?
```
env: YARN_VERSION: 1.22.19
run: npm install --global --force yarn@$YARN_VERSION
```

Reply:
Heh Heh Heh.  Hadn't thought of that, good idea.

Will update the PR shortly. :smile:

Reply:
Just pushed an extra commit with that change.  Seem ok to you? :smile:

Reply:
Yes, it looks good. I've approved it. :)

Reply:
Thanks heaps. :smile: